src/message.h

```diff
@@ -115,12 +115,13 @@ template<class ErrT, class... ArgTs> void error(const ArgTs &... args) {
 #define MSG_TITLE "thinkfan " VERSION ": A minimalist fan control program"
 
 #define MSG_USAGE \
- "Usage: thinkfan [-hnqzD [-b BIAS] [-c CONFIG] [-s SECONDS] [-p [SECONDS]]]" \
+ "Usage: thinkfan [-hnqDd [-b BIAS] [-c CONFIG] [-s SECONDS] [-p [SECONDS]]]" \
```

Comment:
It appears that there is a `-d` option, but only applicable with `USE_ATASMART` is defined- see https://github.com/vmatare/thinkfan/blob/69410/src/thinkfan.cpp#L169
Is that feature deprecated?

Reply:
Oh, you're right. And no, it's not deprecated (yet) but at some point it should be because the libatasmart is unreasonably cpu-hungry and recent kernels have an hwmon interface for S.M.A.R.T. hard disks. Anyways... in that case, this change is correct.